### ISSUE TITLE
[Major] Clarify provenance of sample datasets

### ISSUE THREAD

**Issue by niksirbi** [calebweinreb/SNUB#21]
Thanks for publishing the data on Zenodo @calebweinreb #10.

I couldn't find any information about the provenance of the sample datasets though. If these are taken from public data repositories, I suggest citing them. If these are original data, you would have to mention something about the licensing of animal experiments and confirm that it complies with the [JOSS guidelines on the subject](https://joss.readthedocs.io/en/latest/policies.html#joss-policies). 

I would include the provenance info in the Zenodo data description, and cite the Zenodo dataset in the paper.

**Comment by calebweinreb** [comment]
I have updated the JOSS paper and the Zenodo description to clarify the source of the data used in the tutorial (see https://github.com/calebweinreb/SNUB/commit/4717b600c29a01c3549b1500812e5e81eb9d9bd7). The data are original and as yet unpublished. Here is a list of relevant changes:

- The author list of the JOSS paper and the Zenodo dataset have been updated to include the colleague who collected one of the sample datasets (I collected the other one).
- The JOSS paper notes that the data are original, includes an attestation of ethical compliance, and cites/links the Zenodo dataset
- The Zenodo data description also notes that the data are original, includes an ethical compliance statement, and also has a brief description of the experimental procedures for animal surgery, data collection and post-processing.
